build/Makefile

```diff
@@ -47,7 +47,10 @@ version:
 	@echo $(package_version)
 
 # Run all tests
-test: ensure-build-image
+test: ensure-build-image test-quilkin test-examples
```

Comment:
Split up quilkin tests from examples tests here into separate targets, in case someone only wants to test one at a time.